git/remote.py

```diff
@@ -391,7 +391,7 @@ def __init__(self, repo, name):
 
     def __getattr__(self, attr):
         """Allows to call this instance like
-        remote.special( \*args, \*\*kwargs) to call git-remote special self.name"""
+        remote.special( \\*args, \\*\\*kwargs) to call git-remote special self.name"""
```

Comment:
It seems like you might want to have the docstring be WYSIWYG, and the raw string appears to be meant for that, to me...